.vscode/launch.json

```diff
@@ -1,34 +1,26 @@
 {
-	"version": "0.1.0",
+	"version": "0.2.0",
 	"configurations": [
 		{
 			"name": "Launch Extension",
 			"type": "extensionHost",
 			"request": "launch",
 			// path to VSCode executable
 			"runtimeExecutable": "${execPath}",
-			"args": [
-				"--extensionDevelopmentPath=${workspaceFolder}"
-			],
+			"args": ["--extensionDevelopmentPath=${workspaceRoot}"],
```

Comment:
There was another PR that got merged to master which replaced all `$workspaceRoot` instances with `$workspaceFolder`. 

I believe the net change you want here is to change the version number. Can you undo the other changes in this file?